Source/WebKit/UIProcess/ios/forms/WKFileUploadPanel.mm

```diff
@@ -845,51 +877,28 @@ - (void)presentationControllerDidDismiss:(UIPresentationController *)presentatio
     return WebCore::multipleFileUploadText(urlsCount);
 }
 
-
 - (void)documentPicker:(UIDocumentPickerViewController *)controller didPickDocumentsAtURLs:(NSArray<NSURL *> *)urlsFromUIKit
 {
     ASSERT(urlsFromUIKit.count);
     [self _dismissDisplayAnimated:YES];
 
     dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0), makeBlockPtr([retainedSelf = retainPtr(self), urlsFromUIKit = retainPtr(urlsFromUIKit)] () mutable {
+        ASSERT(!RunLoop::isMain());
```

Comment:
This assertion isn't really useful, it's always under `dispatch_async(dispatch_get_global_queue`.